app/models/payment.rb

```diff
@@ -0,0 +1,8 @@
+class Payment < ActiveRecord::Base
+  validates :name, presence: true, length: { minimum: 4, maximum: 30 }
+  validates :amount, presence: true, numericality: { only_integer: true, other_than: 0 }
+
+  belongs_to :author, class_name: 'User'
+  has_many :operations
+  has_many :categories, through: :operations
```

Comment:
- [ ] Great work implementing this relationship. This is a great interpretation of the ERD diagram you were provided with. 👍

Reply:
Thanks! I was really proud of it, I tried so many ways that didn't worked, so I made a research and found this and another way, but this seems to be more complete